docusaurus/sidebars.js

```diff
@@ -65,9 +65,21 @@ const sidebars = {
             type: 'doc',
             id: 'dev-docs/configurations',
           },
+          
           items: [
-            // 'dev-docs/configurations/database',
-            // 'dev-docs/configurations/server',
+            {type: 'category',
+            label: 'Required configurations',
+            items: [
+              'dev-docs/configurations/server',
```

Comment:
We would need to add all the other files that are included in this PR as well here :)